info.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "dnstwist",
+  "version": "1.0.0",
+  "label": "dnstwist",
+  "description": "Domain name permutation engine for detecting homograph phishing attacks,typosquatting, fraud and brand impersonation.",
+  "publisher": "Fortinet",
+  "icon_small_name": "dnstwist_small.png",
+  "icon_large_name": "dnstwist_large.png",
+  "cs_approved": false,
+  "cs_compatible": true,
+  "category": "investigation",
+  "help_online": "https://github.com/elceef/dnstwist",
+  "configuration": {
+    "fields": [
```

Comment:
this connector doesn't need any configuration, so you can remove fields[]. requirements.txt will have the required packages which will be installed automatically

Reply:
solved